### PR TITLE
[GCU] New 4700 SKU addition to RDMA validator

### PR DESCRIPTION
**Pull request by developfast**
#### What I did
Added a new Mellanox HWSKU to the RDMA GCU validator list - Mellanox-SN4700-O8V48, since RDMA based GCU tests were failing due to invalidation on this HWSKU.

#### How I did it
Added the appropriate hwsku to the GCU list

#### How to verify it
Run the dynamic threshold RDMA GCU test and verify it passes.

### CODE REVIEW

**Comment by developfast** [comment]
@isabelmsft can you pls help review?

**Comment by isabelmsft** [comment]
Please add UT following the example in this PR: https://github.com/sonic-net/sonic-utilities/pull/3038/files

**Review by isabelmsft — APPROVED** [review]

**Comment by stephenxs** [comment]
Hi @developfast  @isabelmsft 
I would like to suggest using this PR https://github.com/sonic-net/sonic-utilities/pull/3131 in which all current SKUs have been added to the list.
Would you please approve and merge it?
In the long run, I would like to suggest collecting such information from the device folder. By doing so, such capability can be added by default once a new SKU has been brought up and we do not need to change the common file here.
Thanks.

File: generic_config_updater/gcu_field_operation_validators.conf.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
                 "spc1": [ "ACS-MSN2700", "ACS-MSN2740", "ACS-MSN2100", "ACS-MSN2410", "ACS-MSN2010", "Mellanox-SN2700", "Mellanox-SN2700-D48C8",
                           "ACS-MSN2700-A1", "Mellanox-SN2700-A1", "Mellanox-SN2700-A1-C28D8", "Mellanox-SN2700-A1-D40C8S8", "Mellanox-SN2700-A1-D44C10", "Mellanox-SN2700-A1-D48C8" ],
                 "spc2": [ "ACS-MSN3800", "Mellanox-SN3800-D112C8" ],
-                "spc3": [ "ACS-MSN4700", "ACS-MSN4600", "ACS-MSN4600C", "ACS-MSN4410", "Mellanox-SN4600C-D112C8", "Mellanox-SN4600C-C64", "Mellanox-SN4700-O8C48" ],
+                "spc3": [ "ACS-MSN4700", "ACS-MSN4600", "ACS-MSN4600C", "ACS-MSN4410", "Mellanox-SN4600C-D112C8", "Mellanox-SN4600C-C64", "Mellanox-SN4700-O8C48", "Mellanox-SN4700-O8V48" ],
```
Comment (stephenxs) [inline generic_config_updater/gcu_field_operation_validators.conf.json:23]: Suggest adding all supported SKUs as a short-term solution.
See https://github.com/sonic-net/sonic-utilities/pull/3131

**Comment by developfast** [comment]
#3131 takes a larger list of approved hwskus, so I am closing this PR

**Comment by isabelmsft** [comment]
> Hi @developfast @isabelmsft I would like to suggest using this PR #3131 in which all current SKUs have been added to the list. Would you please approve and merge it? In the long run, I would like to suggest collecting such information from the device folder. By doing so, such capability can be added by default once a new SKU has been brought up and we do not need to change the common file here. Thanks.

Thanks for the good suggestion, @stephenxs . I agree it would be optimal to derive these mappings dynamically. Do you have a sample for how asic type can be mapped to HwSKU based on device folder? I tried testing by going to the ACS-MSN2100 folder and doing a grep for `spc1` (one of the HwSKU/asic pair onboarded by your PR) but did not come up with anything.

**Pull request closed without merging**